tests/util.go

```diff
@@ -115,3 +128,100 @@ func writeJson(jsonFile string, json string) (string, error) {
 	}
 	return jsonFile, nil
 }
+
+// ReplaceStringInFile replace the old string by the new one.
+// If n < 0, there is no limit on the number of replacements.
+func ReplaceStringInFile(file, old, new string, n int) error {
+	input, err := ioutil.ReadFile(file)
+	if err != nil {
+		return fmt.Errorf("Error while reading file %s: ", file)
+	}
+
+	output := bytes.Replace(input, []byte(old), []byte(new), n)
+
+	if err = ioutil.WriteFile(file, output, 0644); err != nil {
+		return fmt.Errorf("Error while writing file %s", file)
+	}
+	return nil
+}
+
+// DownloadFromUrl reads content from a url to a local file.
+func DownloadFromUrl(url, dest string) error {
+	output, err := os.Create(dest)
+	if err != nil {
+		return fmt.Errorf("Error while creating file %s", dest)
+	}
+	defer output.Close()
+
+	response, err := http.Get(url)
+	if err != nil {
+		return fmt.Errorf("Error while downloading %s", url)
+	}
+	defer response.Body.Close()
+
+	_, err = io.Copy(output, response.Body)
+	if err != nil {
+		return fmt.Errorf("Error while writing to %s", dest)
+	}
+	return nil
+}
+
+// RunVirtctlCommand runs virtctl command with specified args.
+func RunVirtctlCommand(args ...string) (string, error) {
+	if KubeVirtVirtctlPath == "" {
+		var err error
+		KubeVirtVirtctlPath, err = exec.LookPath("virtctl")
+		if err != nil {
+			return "", fmt.Errorf("can not find virtctl binary")
+		}
+	}
+
+	kubeconfig := flag.Lookup("kubeconfig").Value
+	if kubeconfig == nil || kubeconfig.String() == "" {
+		return "", fmt.Errorf("can not find kubeconfig")
+	}
+
+	master := flag.Lookup("master").Value
+	if master != nil && master.String() != "" {
+		args = append(args, "--server", master.String())
+	}
+
+	cmd := exec.Command(KubeVirtVirtctlPath, args...)
+	kubeconfEnv := fmt.Sprintf("KUBECONFIG=%s", kubeconfig.String())
+	cmd.Env = append(os.Environ(), kubeconfEnv)
+
+	stdOutErrBytes, err := cmd.CombinedOutput()
+
+	if err != nil {
+		return string(stdOutErrBytes), err
+	}
+	return string(stdOutErrBytes), nil
+}
+
+// CheckVmiRunningStatus return virtualmachine instance running status.
+// If return false if the virtualmachine instance is not in
+// running status after 10 minutes.
+func CheckVmiRunningStatus(ns, vmiName string, timeout time.Duration) (bool, error) {
```

Comment:
I think we have `Eventually` function, please avoid using busy loop and add sleep function to wait